package.json

```diff
@@ -60,6 +60,7 @@
     "serialised-error": "1.1.3",
     "tough-cookie": "3.0.1",
     "word-wrap": "1.2.3",
+    "xml-formatter": "^2.4.0",
```

Comment:
Please remove the carat from the module version. We always use the fixed version for a package so that it automatically doesn't bump and surprise us in the production. We try to update only when required.